Version updates (#7032)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/release_image_disconn.adoc

In some cases, you need to maintain a custom list of release images when the hub cluster has no Internet connection. You can create your own custom list of release images that are available when creating a cluster. Complete the following steps to manage your available release images while disconnected:

> . While you are on a connected system, navigate to the link:https://github.com/stolostron/acm-hive-openshift-releases/tree/backplane-2.7[acm-hive-openshift-releases GitHub repository] to access the cluster image sets that are available.

Reviewer: Optional: though not a big deal, IBM SG suggests saying "go to" instead of navigate. 

```suggestion
. When you are on a connected system, go to the link:https://github.com/stolostron/acm-hive-openshift-releases/tree/backplane-2.7[acm-hive-openshift-releases GitHub repository] to access the available cluster image sets.
```
